sale_timesheet_invoice_description/readme/USAGE.rst

```diff
@@ -0,0 +1,16 @@
+To use this module, you need to:
+
+#. Go to *Sales -> Orders -> Orders* and create a new Sale Order.
+#. Add line selecting a product with
+
+   - *Product Type* -> **Service**
+   - *Service Invoicing Policy* -> **Timesheets on tasks**
+   - *Service Tracking* -> **Create a task in a new project**
```

Comment:
There is no **Create a task in a new project** option

Reply:
I think the equivalent now in v13 is: **Create a task in sales order's project**

Reply:
yes, you are right, this text in v13 is different